packages/nouns-webapp/src/components/StandaloneNoun/index.tsx

```diff
@@ -93,8 +93,9 @@ export const StandaloneNounWithSeed: React.FC<StandaloneNounWithSeedProps> = (
 
   const dispatch = useDispatch();
   const seed = useNounSeed(nounId);
+  const seedIsInvalid = Object.values(seed || {}).every(v => v === 0);
```

Comment:
Non blocking question:

when would this happen?

Reply:
The aardvark would be shown for 3-5 seconds if sitting on the auction page when the settlement function is called. Users ask about it once in awhile. While this solution isn't perfect, the chance of getting a Noun with seed `[0,0,0,0,0]` is 1 in 40,393,080, so good enough!